src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -10,12 +11,14 @@
 public class DriverManager {
 
 	private Job2dDriver currentDriver = new LoggerDriver();
+	private Publisher driverChange = new Publisher();
```

Comment:
similar changes are already in the repo:
https://github.com/iis-powp-2020/powp_jobs2d_projects/pull/51/files
https://github.com/iis-powp-2020/powp_jobs2d_projects/blob/IO-2-dev/src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java
??? Maybe something is not up to date.